test/e2e/specs/specs-wpcom/wp-calypso-gutenberg-site-editor-tracking-spec.js

```diff
@@ -128,23 +128,39 @@ const exitSiteEditor = async function ( driver ) {
 
 const deleteAll = async function ( driver ) {
 	// Make sure we have posts before trying to delete them
-	const noItems = await driverHelper.isElementLocated( driver, By.css( '#the-list .no-items' ) );
+	const noItemsRowLocator = By.css( '#the-list .no-items' );
+	const noItems = await driverHelper.isElementLocated( driver, noItemsRowLocator );
 	if ( noItems ) {
 		return;
 	}
 
+	const selectAllCheckboxLocator = By.css( '#cb-select-all-1' );
+	// We use the bottom bulk action dropdown and apply button because those are
+	// available on both mobile and desktop.
+	const bottomBulkActionDropdownLocator = By.css( '#bulk-action-selector-bottom' );
+	const bottomBulkActionApplyButtonLocator = By.css( '#doaction2' );
+	const createBottomBulkActionDropdownOptionLocator = ( value ) =>
+		By.css( `#bulk-action-selector-bottom option[value="${ value }"]` );
+	const trashedItemsLinkLocator = By.css( '.subsubsub .trash a' );
```

Comment:
Is `.subsubsub` a valid selector? 😄

Reply:
Yep! That's the bar above the post tabe list where you see the statuses as links 😄